tox_pipenv/plugin.py

```diff
@@ -7,80 +8,61 @@
 import contextlib
 
 
-def _init_pipenv_environ():
-    os.environ["PIPENV_ACTIVE"] = "1"
+ENV_PIPENV_INSTALL_ARGS = "TOX_PIPENV_INSTALL_ARGS"
```

Comment:
nit: I think OPTS is more conventional
```suggestion
ENV_PIPENV_INSTALL_ARGS = "TOX_PIPENV_INSTALL_OPTS"
```
Also, I think the INSTALL bit isn't the clearest since this allows passing any command (e.g. `sync`).

Maybe we should keep it and not require a command (i.e. `TOX_PIPENV_INSTALL_OPTS="--deploy"`).
(I think `sync` is equivalent to `install` without args FWIW.)

Reply:
Ended up splitting this into cmd and opts.

If cmd is specified, then we don't apply in built opts like --pre which might fail.